### PR TITLE
Fix details row cell size

### PR DESCRIPTION
**Pull request by marenar**
#### Description of changes

Make the details row cells stretch to the total height of the row using flexbox.

#### Focus areas to test

Details row cells

### CODE REVIEW

**Comment by betrue-final-final** [comment]
Thanks for doing this!

**Comment by micahgodbolt** [comment]
Flex all the things!

**Comment by micahgodbolt** [comment]
Any chance you can toss in a couple screenshots describing the problem and how this code change fixes it?

**Comment by marenar** [comment]
Sure, @micahgodbolt. Here it is before, background red for visibility. As you can see the details row cell height is not the height of the row, making custom rendering like a different background color for the cell difficult.
details_row_size_not_fixed

And here it is fixed. The details row cell height stretches to the total row height.
details_row_size_fixed

**Comment by micahgodbolt** [comment]
Ooooh yeah. Flexbox to the rescue!

File: packages/office-ui-fabric-react/src/components/DetailsList/DetailsRow.scss [hunk 1]
```diff
@@ -11,7 +11,7 @@ $detailsList-item-selected-hover-color: $listItemSelectedHoverColor;
 .root {
   @include focus-border();
 
-  display: inline-block;
+  display: flex;
```
Comment (micahgodbolt) [inline packages/office-ui-fabric-react/src/components/DetailsList/DetailsRow.scss]: is this flex required? it seems that this should work with just an update to the fields.

Comment (marenar) [inline packages/office-ui-fabric-react/src/components/DetailsList/DetailsRow.scss]: It is yes. Otherwise, the checkmark to the left becomes misaligned.

Comment (micahgodbolt) [inline packages/office-ui-fabric-react/src/components/DetailsList/DetailsRow.scss]: ahh yup :) I "had it working" but didn't realize my checkmark was in the wrong place.

**Pull request merged**